x/mongo/driver/operation.go

```diff
@@ -1116,7 +1116,7 @@ func (op Operation) createLegacyHandshakeWireMessage(
 	wmindex, dst = wiremessage.AppendHeaderStart(dst, info.requestID, 0, wiremessage.OpQuery)
 	dst = wiremessage.AppendQueryFlags(dst, flags)
 
-	dollarCmd := [...]byte{'.', '$', 'c', 'm', 'd'}
+	var dollarCmd = [...]byte{'.', '$', 'c', 'm', 'd'}
```

Comment:
Do we have a preference for `var variable = ...` over `variable := ...` in the convention?

Reply:
I've reverted this change.